cmd/addblock/import.go

```diff
@@ -345,7 +345,7 @@ func newBlockImporter(db database.DB, r io.ReadSeeker) (*blockImporter, error) {
 		IndexManager: indexManager,
 		// No nice way to get the main configuration here.
 		// For now just accept up to the default.
-		ExcessiveBlockSize: 32000000,
+		ExcessiveBlockSize: 32000000 * 4, // TODO TODO, is it needed to do that here really?
```

Comment:
Eek this doesn't look ideal... where'd you get 4? What are other implementations doing?

Reply:
Yeah I agree that it isn't ideal at all. As the comment above it says, there doesn't seem to be a nice way to get the right exact number there. So for now I just kept the old solution. The reason for choosing 2^2 is ABLA state can only grow 2x in like every 6 months. (This number is for the message part and is unrelated to the block validation.) Other implementations have different arch iirc. Not sure what's the best way to use a more precise number. One way to solve this is to introduce some sort of global variable for example.